flake.nix

```diff
@@ -0,0 +1,189 @@
+{
+  description = "Static analyser of semantic differences in large C projects";
+
+  inputs = { nixpkgs.url = "github:NixOS/nixpkgs/release-23.05"; };
+
+  outputs = { self, nixpkgs, ... }:
+    let
+      system = "x86_64-linux";
+      pkgs = import nixpkgs { inherit system; };
+
+      llvmVersionMin = 9;
+      llvmVersionMax = 16;
+      llvmVersions = pkgs.lib.lists.range llvmVersionMin llvmVersionMax;
+
+      mkDiffkemp =
+        llvmPackages:
+          with pkgs;
+          python3Packages.buildPythonPackage {
+            pname = "diffkemp";
+            version = "0.5.0";
+
+            src = self;
+
+            nativeBuildInputs = with llvmPackages; [
+              cmake
+              gcc
+              libllvm
+              ninja
+              nodejs_20
+            ];
+
+            buildInputs = with llvmPackages; [
+              clangNoLibcxx
+              cscope
+              diffutils
+              gtest
+              gnumake
+            ];
+
+            propagatedBuildInputs = with python3Packages; [
+              cffi
+              pyyaml
+              setuptools
+            ];
+
+            WITHOUT_RPYTHON = true;
+
+            # Including cmake in nativeBuildInputs automatically runs it during
+            # configurePhase so we just need to set correct flags.
+            cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Debug" "-GNinja" "-DBUILD_VIEWER=On" ];
```

Comment:
```suggestion
            cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Debug" "-GNinja" ];
```

Reply:
Good catch!

Agreed - let's disable the viewer in build for now to unblock this and come back to it later.